pkg/workloadmeta/collectors/internal/remote/process_collector/process_collector.go

```diff
@@ -113,15 +114,15 @@ func init() {
 	workloadmeta.RegisterCollector(collectorID, func() workloadmeta.Collector {
 		return &remote.GenericCollector{
 			CollectorID:   collectorID,
-			StreamHandler: &streamHandler{},
+			StreamHandler: &streamHandler{Config: config.Datadog},
```

Comment:
whould we pass an new instance of the `config.Config{}` instead of pointing to the global variable `config.Datadog` ?